tests/integration/crud/advanced/yaml/rank.yml

```diff
@@ -0,0 +1,7 @@
+!MinRanker
```

Comment:
this has been deprecated, and please use some MockExecutors to test, to keep independency between core and hub